nominees/open-referral.json

```diff
@@ -0,0 +1,55 @@
+    },
+    {
+      "SDGNumber": 3,
+      "evidenceText": "Every community faces a similar challenge: there are many different kinds of health, human, and social services that are available to people in need, yet no one way that information about them is produced and shared. Instead, many organizations collect and structure community resource directory data in different ways — yielding redundant, fragmented silos.  As a result, it’s hard to ‘see’ the safety net. Many people never discover services that could help improve their lives. Service providers spend precious time verifying data rather than helping people. And without access to this information, decision-makers struggle to evaluate community health and program effectiveness. This yields underperforming systems that fail people and communities in tragic ways."
+    },
+    {
+      "SDGNumber": 10,
+      "evidenceText": "Every community faces a similar challenge: there are many different kinds of health, human, and social services that are available to people in need, yet no one way that information about them is produced and shared. Instead, many organizations collect and structure community resource directory data in different ways — yielding redundant, fragmented silos.  As a result, it’s hard to ‘see’ the safety net. Many people never discover services that could help improve their lives. Service providers spend precious time verifying data rather than helping people. And without access to this information, decision-makers struggle to evaluate community health and program effectiveness. This yields underperforming systems that fail people and communities in tragic ways."
+    },
+    {
+      "SDGNumber": 16,
+      "evidenceText": "Open Referral makes it easier for many institutions to collaborate on the management of information about the resources available to people in need – which can enhance partnerships, needs assessment, program evaluation, and institutional design. "
+    },
+    {
+      "SDGNumber": 17,
+      "evidenceText": "Open Referral makes it easier for many institutions to collaborate on the management of information about the resources available to people in need – which can enhance partnerships, needs assessment, program evaluation, and institutional design. "
+    }
+  ],
+  "sectors": [],
+  "type": [
+    "standard"
+  ],
+  "repositoryURL": "",
```

Comment:
We should link to the repository that contains the actual standard: https://github.com/openreferral/specification

Reply:
Resolved in commit bed6e91